doc/application_workflow/webserving.md

```diff
@@ -0,0 +1,137 @@
+# Web-serving on vSphere Integrated Container Engine
+
+We take the Web-serving benchmark from CloudSuite (http://cloudsuite.ch/webserving/) as an example, to demonstrate how customers who are interested in the LEMP implementation of a cloud native web-serving application could deploy it on vSphere Integrated Container Engine 0.7.0 with Docker Compose. This demo has three tiers deployed on three containerVMs: an Nginx Web server, a Memcached server, and a MySQL database server. The Web server runs Elgg (a social networking engine) and connects the Memcached server and the database server through the network.
+
+## Workflow
+
+### Build docker image for the Web server (on regular docker)
+
+In the original the Web-server docker image from Cloudsuite, the functionality of “email verification for new user registration” is not enabled, which makes it less realistic and practical. Therefore, we need make some modifications and re-build the docker image for the Web server. **You can also skip this section and proceed to "Compose File for vSphere Integrated Containers Engine" if you do not want to build your own image**.
```

Comment:
* Did you mean `In the original Web-server docker image`?
* You can link directly to the `Compose File for vSphere Integrated Containers Engine` section using markdown with `#compose-file-for-vsphere-integrated-containers-engine`